lib/xenops_server.ml

```diff
@@ -1919,12 +1933,13 @@ and perform_exn ?subtask ?result (op: operation) (t: Xenops_task.task_handle) :
           let vgpus' = VGPU_DB.vgpus id in
           let pcis_sriov = List.filter (is_nvidia_sriov vgpus') pcis in
           List.concat
-            [ List.map (fun pci -> PCI_plug(pci.Pci.id, false)) pcis_sriov
+            [ dequarantine_ops vgpus' pcis
+            ; List.map (fun pci -> PCI_plug(pci.Pci.id, false)) pcis_sriov
             ; [VGPU_start (vgpus, true)]
             ]
         in
         perform_atomics (List.concat
-          [ List.map (fun pci -> PCI_dequarantine pci.Pci.id) pcis
+          [ List.map (fun pci -> PCI_dequarantine pci.Pci.address) pcis
```

Comment:
Isn't this covered by the `dequarantine_ops` above?

Reply:
Yes. This line needs to go.